addons/website_slides/controllers/mail.py

```diff
@@ -15,12 +16,18 @@ class SlidesPortalChatter(PortalChatter):
 
     @http.route(['/mail/chatter_post'], type='json', methods=['POST'], auth='public', website=True)
     def portal_chatter_post(self, res_model, res_id, message, **kw):
+        previous_post = request.env['mail.message'].search([('res_id', '=', res_id),
+                                                            ('author_id', '=', request.env.user.partner_id.id),
+                                                            ('model', '=', 'slide.channel'),
+                                                            ('subtype_id', '=', request.env.ref('mail.mt_comment').id)])
+        if previous_post:
+            raise ValidationError("Only a single review can be posted per course.")
```

Comment:
is this error handled properly by the frontend ? (it might be visible for the user, so it should be translated)